.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -56,6 +56,7 @@ Note: Adding GitHub labels is only possible for contributors with write access.
 
 - [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
 - [ ] The appropriate `team/..` label has been applied, if known.
+- [ ] Use the `major_change` label if your change either has a major impact, is impacting multiple teams or is changing important well-established internals of the Agent.
```

Comment:
I think we should quickly explain that this is not for customer facing but for QA purposes. I think the difference between breaking changes and `majort_change` is important here. WDYT ?

Reply:
Would you prefer something like:

`For QA purposes, use the `major_change` label if ...`

I agree this is not really intended for external contributor but I hardly see one making such a big change that he or she would have to cross this box.

Reply:
I agree and an external contributor can't set label anyway.

WDYT:
 ```suggestion
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
```